<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpManual</class>
 <widget class="QDialog" name="HelpManual">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open RPG Maker - Help Manual</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabContents">
       <attribute name="title">
        <string>Contents</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="twContents">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="indentation">
           <number>15</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabIndex">
       <attribute name="title">
        <string>Index</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Look for:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leIndexSearch"/>
        </item>
        <item>
         <widget class="QListWidget" name="lwIndexList">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="bBack">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bForward">
           <property name="text">
            <string>Forward</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWebView" name="webView">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HelpManual</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HelpManual</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBack</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bForward</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
